pkg/controlplane/funcs.go

```diff
@@ -0,0 +1,21 @@
+package controlplane
+
+// GetFencingRoles - roles that normally require fencing
+func GetFencingRoles() []string {
+	// Can't create an array constant, so this is what we'll do instead
+	return []string{
+		"CellController",
+		"ControllerAllNovaStandalone",
+		"ControllerNoCeph",
+		"ControllerNovaStandalone",
+		"ControllerOpenstack",
+		"ControllerSriov",
+		"ControllerStorageDashboard",
+		"ControllerStorageNfs",
+		"Controller",
+		"Database",
+		"Messaging",
+		"Standalone",
+		"Telemetry",
```

Comment:
we could also get information from the default roles from inside the openstackclient image from roles in /usr/share/openstack-tripleo-heat-templates/roles

Reply:
That would be cool.  What would be the approach?  I'm just wondering what you had in mind for how the operator would access the data inside the openstackclient image or pod.

Reply:
When the ctlplane CR gets created we can run an init job, which gets OSP information from the openstackclient image into into a CM. the controllers which need default ooo information could then parse the information from the cm.

Reply:
Alright, let's also do this in a follow-up PR if you would be okay with that

Reply:
ack